contrib/mesos/pkg/minion/task.go

```diff
@@ -0,0 +1,519 @@
+/*
```

Comment:
It's much easier to identify flaws in the abstractions if one tries to move them into their own module. Moreover, it increases the readability because it makes public/private explicit and automatically checked by the compiler. And it doesn't cost anything.